app/DoctrineMigrations/Version20230807105004.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230807105004 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE tour ADD date DATE');
```

Comment:
Should be: 

```
$this->addSql('ALTER TABLE tour ADD date DATE NOT NULL');
```

In this case, for the migration to work, you would need to first add it as DEFAULT NULL, then update data, and finally change it to NOT NULL, like in [this migration](https://github.com/coopcycle/coopcycle-web/blob/4b894429515216b79a4b9fa2dea7bcd4047d3192/app/DoctrineMigrations/Version20231025114558.php).